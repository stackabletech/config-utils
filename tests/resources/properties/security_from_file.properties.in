networkaddress.cache.negative.ttl=0
networkaddress.cache.ttl=${file:UTF-8:/etc/hostname}
meine.suß.Pröpertie=42,3
