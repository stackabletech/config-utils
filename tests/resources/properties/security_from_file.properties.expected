networkaddress.cache.negative.ttl=0
networkaddress.cache.ttl=nixos
meine.suß.Pröpertie=42,3
